test/config.js

```diff
@@ -7,8 +7,10 @@ const gitFixture = require('./fixtures/git')
 
 const wiby = require('..')
 
+const nodeVersion = Number(process.versions.node.split('.')[0])
```

Comment:
```suggestion
const semver = require('semver');
const isGTE20 = semver.satisfies(process.version, '>= 20');
```
to be more reliable?

Reply:
It would mean installing another dependency for something so simple, and since it's only used in this part, I don't think adding that dependency is justified.

Reply:
There’s zero cost to adding extra dev deps, and i think doing a guaranteed-correct check seems worth it ¯\_(ツ)_/¯

Reply:
done